src/app/search/advancedsearch/advancedsearch.component.scss

```diff
@@ -13,3 +13,12 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
+
+.dialog-header {
+  padding: 15px;
```

Comment:
I'll beat Gary to this and say "don't use pixels".

Reply:
Removed the scss class since it wasn't being used